src/main/java/org/spongepowered/common/config/category/ModuleCategory.java

```diff
@@ -30,32 +30,33 @@
 @ConfigSerializable
 public class ModuleCategory extends ConfigCategory {
 
-    @Setting(value = "bungeecord")
+    @Setting(value = "bungeecord", comment = "???")
     private boolean pluginBungeeCord = false;
 
-    @Setting(value = "entity-activation-range")
+    @Setting(value = "entity-activation-range", comment = "???")
```

Comment:
Entity activation module, allows you to tweak entity activation ranges, similarly to "Entity Distance" in the latest 1.16 snapshots but per mob. Essentially it changes how far an entity can be away from a player, until it stops being updated (e.g. moving).